android/java/res/layout/brave_rewards_site_banner.xml

```diff
@@ -36,4 +36,15 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <ImageView
+        android:id="@+id/button_close"
+        android:layout_width="30dp"
+        android:layout_height="30dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="20dp"
+        android:src="@drawable/ic_baseline_close_24"
```

Comment:
We should use app:srcCompat else it would crash on Android 6

Reply:
used both android:src=  and app:srcCompat

Reply:
> used both android:src=  and app:srcCompat
> 

Actually we don't need to use android:sec at all as app:srcCompat also supports  backward compatible.

Reply:
https://github.com/chromium/chromium/blob/main/chrome/browser/notifications/android/java/res/layout/notification_permission_rationale_dialog.xml